web/api/v6/report_server.thrift

```diff
@@ -222,6 +222,11 @@ struct ReportData {
 }
 typedef list<ReportData> ReportDataList
 
+struct BugPathLengthFilter {
```

Comment:
I thinking to rename it to `BugPathLengthRange`, do we plan to extend this struct in the future with additional elements?